### PR TITLE
Short answer for all windows (ksqlDB)

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

Continuation of https://github.com/confluentinc/kafka-tutorials/issues/600

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-600-windows/
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-600-windows/create-tumbling-windows/ksql.html
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-600-windows/create-session-windows/ksql.html
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-600-windows/create-hopping-windows/ksql.html

### CODE REVIEW

**Review by awalther28 — APPROVED** [review]
LGTM

File: _includes/tutorials/session-windows/ksql/markup/answer/answer.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Create a `TABLE` with the `WINDOW SESSION` syntax, and specify the window duration with the parantheses.
```
Comment (bbejeck) [inline _includes/tutorials/session-windows/ksql/markup/answer/answer.adoc:1]:
```suggestion
Create a `TABLE` with the `WINDOW SESSION` syntax, and specify the maximum inactivity period defining a session with the parantheses.
```

**Review by bbejeck — COMMENTED** [review]
Thanks for this PR @ybyzek overall looks great, I have one small comment.

**Review by bbejeck — APPROVED** [review]
Thanks for the quick changes @ybyzek LGTM.

**Pull request merged**